acct-user/headscale/headscale-0-r1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit acct-user
+
+DESCRIPTION="Headscale Server hosting user"
+
+ACCT_USER_ID=600
+ACCT_USER_HOME_OWNER=headscale:headscale
+ACCT_USER_HOME_PERMS=750
+ACCT_USER_SHELL=/bin/sh
+ACCT_USER_GROUPS=( headscale )
+
+
```

Comment:
two blank lines (should eliminate all, or single line)

common issue, no more comment for later review